src/features/bulk/FileViewer/FileViewer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { DocumentViewer } from '@amsterdam/bmi-component-library';
+import debounce from 'debounce';
+
+import { MetadataGenericType } from '../../../types';
+import { useAppSelector } from '../../hooks';
+import { getChangeIndividualFields, getDefaultFields, getFields } from '../bulk/store/selectors';
+import { IBulkFile } from '../bulk/store/model';
+import { TGetDocumentViewUrl } from '../bulk/types';
+
+import DefaultFieldsTable from './DefaultFieldsTable';
+import IndividualFieldsForm from './IndividualFieldsForm';
+import {
+	FileViewerDocumentInnerStyle,
+	FileViewerDocumentStyle,
+	FileViewerFieldsStyle,
+	FileViewerStyle,
+} from './styles';
+import { DEFAULT_DEBOUNCE } from '../bulk/constants';
+
+export type Props = {
+	file: IBulkFile;
+	getDocumentViewUrl: TGetDocumentViewUrl;
+	onChange: (data: MetadataGenericType, valid: boolean) => void;
+};
+
+export default function FileViewer(props: Props) {
+	const { file, getDocumentViewUrl, onChange } = props;
```

Comment:
Can be safely destructured on the previous line.

Reply:
check